posts/peachpellen/5_Apps_for_Android.jade

```diff
@@ -0,0 +1,56 @@
+---
+title:      '5 Apps for Android Developers'
+categories: [ideate, launch, design, build]
+heroimage:  ''
```

Comment:
Hey Peach - can you also update your categories line (line 3)? We will want to classify your post under 1 of the 4 categories.